### PR TITLE
fix: several things in frontend

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [x] Adjust db upgrade & data migration messages
- [x] Address migration will wait until user are really logged in

### CODE REVIEW

**Comment by kelsos** [comment]
So what I see when I check out and run locally is the following

image

At least the second error is because of `notify` which for some reason is undefined

File: frontend/app/src/store/blockchain/accounts/migrate.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    watch(logged, logged => {
+      checkLoggedStatus(logged);
+    });
+
+    return {
+      migratedAddresses,
+      upgradeMigratedAddresses
+    };
+  }
+);
```
Comment (kelsos) [inline frontend/app/src/store/blockchain/accounts/migrate.ts:99]: missing the hot-reload part at the end of the store.

File: frontend/app/src/store/blockchain/accounts/migrate.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  type Notification,
+  NotificationCategory,
+  Severity
+} from '@rotki/common/lib/messages';
+import { type MigratedAddresses } from '@/types/websocket-messages';
+import { useSessionAuthStore } from '@/store/session/auth';
+import { startPromise } from '@/utils';
+import { useNotificationsStore } from '@/store/notifications';
+
+export const useAccountMigration = defineStore(
```
Comment (kelsos) [inline frontend/app/src/store/blockchain/accounts/migrate.ts:12]: can we rename it to `useAccountMigrationStore`?

I have a template I created in WebStorm to generate the template for a store.
It still requires some manual interaction in nested ones but I can share it if you want.

**Comment by LefterisJP** [comment]
> So what I see when I check out and run locally is the following
> 
> image
> 
> At least the second error is because of `notify` which for some reason is undefined

I also see this.

**Review by LefterisJP — COMMENTED** [review]
Another thing I see is 



As I said in discord it should say "Data migration 8 is running". Not 8 - 8.

File: frontend/app/src/locales/en.json [hunk 3]
```diff
@@ -1003,10 +1003,10 @@
     "newly_detected": {
       "accept_selected": "Accept selected tokens",
       "ignore_selected": "Ignore selected tokens",
+      "seen_during_or_tx_origin": "Seen during / Tx origin",
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:1006]:
```suggestion
      "seen_during": "Seen during",
```

**Review by LefterisJP — COMMENTED** [review]
This should be a simple "Seen during" as per the review. "Seen during" easily applies also to tx_hashes

File: frontend/app/src/store/blockchain/accounts/migrate.ts (hunk 2 above)
Comment (lukicenturi) [reply frontend/app/src/store/blockchain/accounts/migrate.ts:12]: sure I can try the template

**Comment by lukicenturi** [comment]
> Another thing I see is
> 
> 
> 
> As I said in discord it should say "Data migration 8 is running". Not 8 - 8.

Updated

File: frontend/app/src/components/status/notifications/NotificationPopup.vue [hunk 4]
```diff
@@ -20,22 +20,24 @@ const dismissAll = async () => {
...
+      set(visibleNotification, next);
+    });
+  }
+};
+
+watch(queue, checkQueue, { deep: true });
+
+onMounted(() => {
+  checkQueue();
+});
```
Comment (lukicenturi) [inline frontend/app/src/components/status/notifications/NotificationPopup.vue:40]: This should fix the issue where the popup of notification for the evm address migration, appear late.

**Pull request merged**